webBrow/index.js

```diff
@@ -0,0 +1,326 @@
+(function() {
+  function Calculator(domElement) {
+    this.domElement = domElement,
+    this.displayNode = this.domElement.querySelector('.calculator-valueDisplay')
+    this.first = 0,
+    this.second = 0,
+    this.operand = '',
+    this.result = '',
+    this.firstValueAdded = false,
+    this.secondValueAdded = false
+    this.addTheListeners()
```

Comment:
Exposing ignorance. What does this do? Also, the calculator is pre-loading operators.